mapdata/WarcraftLegacies/UnitData/Skin/1093677125.json

```diff
@@ -90,7 +82,7 @@
     {
       "Id": 1651864693,
       "Type": 3,
-      "Value": "Ancient bear hero, adept at frontline melee combat.|n|cfff5962dAbilities:|r Cleaving Attack|n|cfff5962dCan Learn:|R Maiming Strike, War Stomp, Endurance Aura, Avatar|n|n|cffffcc00Attacks land units.|R"
+      "Value": "Ancient bear hero, adept at frontline melee combat.|n|cfff5962dAbilities:|r Heavy Bash|n|cfff5962dCan Learn:|R Maiming Strike, Wild Toughness, Endurance Aura, Avatar|n|n|cffffcc00Attacks land units.|R"
```

Comment:
Why Wild Toughness? This is mechanically identical to Toughness so it should have the same name and icon. The values are even identical to Rexxar's version.

Reply:
Its Naisha's spell

Reply:
They are on different hotkey i think

Reply:
Right, bugger. Really all 3 heroes should just have Toughness - being on a different hotkey means nothing. But that's not related to this change so that's fine.

Reply:
il set them all to toughness